### DISCUSSION TITLE
install tailwind elements

### DISCUSSION THREAD

**Discussion by Deepakpradhan7** [mdbootstrap/TW-Elements#1435] [Support from community]
i am making a website in next js using tailwind css.After installation i don't know where and how to add script file in the project.
i am putting it in app.js file

app.js 
<script src="../node_modules/tw-elements/dist/js/index.min.js"></script>
        <Script src="https://cdn.jsdelivr.net/npm/tw-elements/dist/js/index.min.js"></Script>   
      <Component {...pageProps} />

**Comment by Trochonovitz** [comment]
Hi, all you got to do is downloading Tailwind Elements from NPM and then import library in _app.js file using useEffect hook - like this. FYI in next week we are going to release a bunch of tutorials for most popular CRS and SSR frameworks so stay tuned.
```
  useEffect(() => {
    const use = async () => {
      (await import("tw-elements")).default;
    };
    use();
  }, []);
```

**Reply by Deepakpradhan7** [reply]
getting this error when i try to import it although i have already installed tw-elements and it is present inside node_modules

Could not find a declaration file for module 'tw-elements'. 'c:/Users/deepa/OneDrive/Documents/GitHub/typof-react/node_modules/tw-elements/dist/js/index.min.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/tw-elements` if it exists or add a new declaration (.d.ts) file containing `declare module 'tw-elements';`

**Reply by Trochonovitz** [reply]
This is not an error but a warning - it should not affect program. And yes - you can import a library only when you already installed.

**Reply by Deepakpradhan7** [reply]
then what might be the reason that the elements are not working properly?

**Reply by Deepakpradhan7** [reply]
all components are working fine excluding dynamic components like carousel, testimonial

**Reply by Trochonovitz** [reply]
Did you edit the tailwind.config.js as instructed on the [Get started page](https://tailwind-elements.com/docs/standard/getting-started/quick-start/)? Is there any other errors/warning beside this one about types? Add to your root directory file typings.d.ts and put there `declare module 'tw-elements'` - this should works.
